tutorials/hana-clients-choose-hana-instance/hana-clients-choose-hana-instance.md

```diff
@@ -175,14 +170,20 @@ The following steps will demonstrate connecting to and examining a SAP HANA, exp
 
     Notice that the database being connected to is SYSTEMDB.
 
-    The example above uses localhost since hdbsql is running on the same machine that the database is trying to connect to.  It is important to note the IP address or host name of the machine, as this will be needed in later tutorials.  The host name `hxehost`, is seen in the terminal above.  
+    The example above uses localhost since hdbsql is running on the same machine that the database is trying to connect to.  The host name `hxehost`, is seen in the terminal above.  
```

Comment:
```suggestion
    The example above uses localhost since hdbsql is running on the same machine that the database is trying to connect to.  The host name, `hxehost`, is seen in the terminal above.  
```